demo/grid-columns-demos.html

```diff
@@ -114,26 +114,23 @@ <h3>Hiding Columns</h3>
     </p>
     <vaadin-demo-snippet id='grid-columns-demos-hiding-columns'>
       <template preserve-content>
-        <vaadin-checkbox>Hide First Two Columns</vaadin-checkbox>
-
+        <vaadin-context-menu>
+          <vaadin-button aria-label="Hide column menu">
```

Comment:
let's add `theme="icon"` to the button so the icon would be aligned nicely.

Reply:
Done